# Content


## About 

I specialize in bioinformatics research related to the construction and use of protein interaction maps.   I have over 17 years of experience building and using bioinformatics applications and leading both small and large development teams.

The last release of the iRefIndex database of consolidated protein interactions is freely available at irefindex.org.

Ian Donaldson
ian@donaldsonresearch.com

[Research Gate Profile](https://www.researchgate.net/profile/Ian_Donaldson2/)  
[LinkedIn Profile](http://www.linkedin.com/pub/ian-donaldson/4/7b8/a86)  
[ResearcherId Profile](http://www.researcherid.com/rid/A-2867-2009)

## Past experience

### Adaptimmune

Presently, I am working at [Adaptimmune](http://www.adaptimmune.com/our-company/overview) as a bioinformatician.  This is a UK biotech company working on a novel and promising form of T-cell therapy to treat cancer.

### Barts and the London Genome Centre

I moved to London in 2015 and was very grateful to get the chance to get some hands-on experience with next-generation sequence data while working with this great  group of people.  I worked on a pipeline for  processing and analyzing single-cell RNASeq data as well as a number of other data types from DNA sequencing, bisulphite sequencing, and short-read sequencing of miRNAs.    Tools used on a day-to-day basis  included Bash, Sun Grid Engine and R.   Some code can be found [here](https://github.com/iandonaldson).

### University of Oslo

During 2005 to 2013, I ran my own research group at the University of Oslo.  This work focused mainly on integration of protein interaction data from multiple primary databases and towards the end, we used these data to investigate their use in identifying drug-targets and [genes involved in multi-locus diseases](http://www.biomedcentral.com/1471-2105/14/47).

The highlight of this work was a series of papers on the [iRefIndex database](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-405) and the production of a series of tools that we used to carry out and share our work.  These included an [R statistics package](http://www.biomedcentral.com/1471-2105/12/455) and a [Cytoscape plugin](http://www.biomedcentral.com/1471-2105/12/388) for visualization of protein interaction data.

Most importantly, this work allowed us to closely examine the use of curation and data exchange standards across multiple databases and quantified [just how important these standards are](http://www.nature.com/nbt/journal/v29/n5/full/nbt.1867.html).

### Blueprint Initiative

Between 2002 and 2005 I worked at the Samuel Lunenfeld Research Institute as part of the Blueprint Initiative.  This project involved over 100 people that worked to build the Biomolecular Interaction Network Database.  Here, I continued to work on project management as well as designing new modules for an in-house bioinformatics platform (called SeqHound) that supported the BIND database.  After this initiative ended, both database systems were bought and are now operated by Thomson as the [BOND database](https://web.archive.org/web/20070908145214/http://bond.unleashedinformatics.com:80/).

### MDS Proteomics

I first cut my teeth on project management at MDS Proteomics under the kind tutelage of an IBM consultant on the job while I planned and managed a group of eight developers that worked on an in-house bioinformatics platform.

MDSP grew at an explosive rate but unfortunately imploded before going public.  However, we did manage to push out one of the [first protein interactomes for yeast](http://www.nature.com/nature/journal/v415/n6868/full/415180a.html).  My small contribution to this work was to estimate the interactions that were already known in the literature using a text-mining system I had written during my post-doctoral work (PreBIND).

### Postdoctoral work

My post-doctoral work was a departure from my Ph.D. in molecular biology and was largely motivated as a means to escape the cold-room.

PreBIND was my first program.  Written in C (including the web-page api) and making use of the NCBI toolkit, this program  collected co-occurrences of protein/gene mentions in pubmed abstracts and assessed the likelihood that they contained interaction data using a support vector machine.  The study was [one of the first to quantify the time savings of computer-assisted curation of biomedical literature](http://www.biomedcentral.com/1471-2105/4/11).

### Ph.D.

This [thesis](https://tspace.library.utoronto.ca/bitstream/1807/12499/1/NQ45748.pdf) now seems buried in the murky past but involved countless hours in a cold-room, a myriad tooth-picks and way too many agar plates.  Mostly forgettable but taught me the importance of never giving up and asking a good question (not necessarily in that order of importance).  Writing my thesis raised the question, "Why aren't all these genetic/physical interactions written down somewhere in a structured-data format that could be easily retreived...?".

## Shared Notes  

* [Regression](https://iandonaldson.github.io/notes/regression/regression.html)
* [Bash](https://iandonaldson.github.io/notes/codingInBashAwkSedUnixAndOsx_notesAndLearningResources.html)
* [R](https://iandonaldson.github.io/notes/codingInR_notesAndLearningResources.html)

## Shared Projects

* [Practical machine learning project ](http://iandonaldson.github.io/practical-machine-learning-project/)




Editing:  see the README for the project [iandonaldson.github.io] (https://github.com/iandonaldson/iandonaldson.github.io)
